sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -573,6 +575,24 @@ final class DataFrameWriter[T] private[sql](ds: Dataset[T]) {
     format("csv").save(path)
   }
 
+  private def runCommand(session: SparkSession, name: String)(command: LogicalPlan): Unit = {
+    val qe = session.sessionState.executePlan(command)
+    try {
+      qe.executedPlan.foreach { plan =>
+        plan.resetMetrics()
```

Comment:
just realized that, in this code path, we pass in a logical plan and will always get a new physical plan, so we don't need to reset metrics here, let me remove it